src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -60,4 +72,79 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
+
+    /**
+     * Stores the details to add a card without a deck assigned.
+     */
+    public static class AddCardDescriptor {
```

Comment:
I have not used this class yet. 

Currently in AddCommandParser, a card is initialized with deck as null. I propose we make a class called CardDescriptor to pass the card arguments to the AddCommand class, where it be initialized with the selectedDeck. (This approach is similar to the EditCommandParser)